FWCore/ParameterSet/python/Modules.py

```diff
@@ -252,6 +252,12 @@ def __init__(self, caseFunctionDict, **kargs):
         self.__setParameters(kargs)
         self._isModified = False
 
+    def setLabel(self, label):
+        super().setLabel(label)
+        for case in self.parameterNames_():
```

Comment:
maybe a comment saying why this is needed?

Reply:
Added comment